rebar.config

```diff
@@ -33,6 +46,9 @@
             {config, "config/test.config"}
         ]}
     ]},
+    {docs, [
```

Comment:
The macro is called `DOC` and the profile used in the alias is `doc` so this should be `doc` too, right?

```suggestion
    {doc, [
```

Reply:
Absolutely - fixed.